aten/src/ATen/core/jit_type.h

```diff
@@ -913,6 +915,7 @@ struct CAFFE2_API DictType : public Type {
       case TypeKind::IntType:
       case TypeKind::BoolType:
       case TypeKind::FloatType:
+      case TypeKind::ComplexDoubleType:
```

Comment:
Supporting complex as a key type for dictionaries some additional changes: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/runtime/register_prim_ops.cpp#L1317

Feel free to remove this and handle it in a follow-up PR.

Reply:
okay removed!